templates/repo/settings/options.tmpl

```diff
@@ -23,20 +23,20 @@
 				<div class="inline field">
 					<label>{{.locale.Tr "repo.template"}}</label>
 					<div class="ui checkbox">
-						<input name="template" type="checkbox" {{if .Repository.IsTemplate}}checked{{end}}>
-						<label>{{.locale.Tr "repo.template_helper"}}</label>
+						<input id="template" name="template" type="checkbox" {{if .Repository.IsTemplate}}checked{{end}}>
+						<label for="template">{{.locale.Tr "repo.template_helper"}}</label>
```

Comment:
Can we nest the `<input>` inside the label in these cases without breaking the rendering? If yes, it would be preferred as that method does not introduce new ids while being equally accessible.

Reply:
Hi.
The rendering remains (almost) the same, but the functionality is totally broken because the initialisation [can't find the `<input>`](https://github.com/go-gitea/gitea/blob/main/web_src/fomantic/build/semantic.js#L1243). Sorry, it was the first approach I tried.

[Linked labels](https://fomantic-ui.com/modules/checkbox.html#linked-labels) is the only option to make foomantic checkbox accessible.

Reply:
I see, `.children` won't work recursively, so it's not a solution that can work.